etl/churn.kp/knowledge.md

```diff
@@ -11,15 +11,15 @@ tags:
 - firefox desktop
 - main_summary
 created_at: 2016-03-28 00:00:00
-updated_at: 2017-03-23 09:50:22.610144
+updated_at: 2017-03-29 15:10:12.039749
 tldr: "Compute churn / retention information for unique segments of Firefox \nusers\
   \ acquired during a specific period of time.\n"
 ---
 # Firefox Destop Churn / Retention Cohort analysis
 
 Compute churn / retention information for unique segments of Firefox users acquired during a specific period of time. Tracked in [Bug 1226379](https://bugzilla.mozilla.org/show_bug.cgi?id=1226379). The underlying dataset is generated via the [telemetry-batch-view](https://github.com/mozilla/telemetry-batch-view/blob/master/src/main/scala/com/mozilla/telemetry/views/MainSummaryView.scala) code, and is generated once a day.
 
-The aggregated churn data is updated weekly. Due to the client reporting latency, we needs to wait 10 days for the data to stabilize. The report will generate a dataset for the closest sunday before `today - 17 days`. The 17 day figure is calculated by finding the closest day we can aggregate data from (10 days), and then find the closest sunday the week before that (at least 7 days). 
+The aggregated churn data is updated weekly. Due to the client reporting latency, we needs to wait 10 days for the data to stabilize. If the date is passed into this report through the environment, it is assumed that the execution date is at least a week greater than the report start date.
```

Comment:
Can you please add an example of the expected date configuration / run info?

Please also change it to "we need to wait"